docs/ScheduledTransactionFrequency.md

```diff
@@ -0,0 +1,7 @@
+# YNAB::ScheduledTransactionFrequency
+
+## Properties
+
+| Name | Type | Description | Notes |
+| ---- | ---- | ----------- | ----- |
+
```

Comment:
Does the ruby generator not output the enum values?

Reply:
No, unfortunately not.  I believe it's not supported in OpenAPI Generator for Ruby (yet).

Reply:
:whomp: :whomp: